modules/op-release-notes-1-9.adoc

```diff
@@ -373,3 +373,42 @@ spec:
 * Before this update, a pull request failed if the user in the annotation provided values by using a regex form, for example, `refs/head/rel-*`. The pull request failed because it was missing `refs/heads` in its base branch. With this update, the prefix is added and checked that it matches. This resolves the issue and the pull request is successful.	
 // Savita Ashture
 
+[id="release-notes-1-9-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.9.1
+
+With this update, {pipelines-title} General Availability (GA) 1.9.1 is available on {product-title} 4.11 and later.
+
```

Comment:
We need to add a known issue section to add this issue https://issues.redhat.com/browse/SRVKP-2854

The value for `CATALOG_REFRESH_INTERVAL`  provided by the user in Tekton Hub CR is not getting updated in the Hub API ConfigMap and the default value of 30m is used by Tekton Hub

Reply:
Hey @PuneetPunamiya - Could you also tell me if we have a workaround for this user too? In the known issues section, we advise the user by giving a workaround as well, if possible.

Reply:
@Dhruv-Soni11 we do not need any workaround here because the default value is 30m so the API will refresh after 30m , is that correct @PuneetPunamiya

Reply:
Thanks for letting me know. I have made the changes as you suggested @rupalibehera .

Reply:
- Workaround
  - update `tekton-hub-api` Configmap from the targetNamespace manually with your changes i.e. update the value for `CATALOG_REFRESH_INTERVAL` and delete the existing `tekton-hub-api-xxx` pod from the targetNamespace

Reply:
Also we need to add a note that

With any update/upgrade the config map will be updated using default values

Reply:
@Dhruv-Soni11 lets merge the PR we can add this info in future releases